### ISSUE TITLE
PresentMon.exe - Entry Point Not Found

### ISSUE THREAD

**Issue by KlimBlaze** [GameTechDev/PresentMon#147]
Hi,

I'm getting the error:

The procedure entry point __stdcalloc_crt could not be located in the dynamic link library C:\ProgramFiles\Intel\PresentMon\PresentMonApplication\PresentMon.exe.

Can't launch app, can anyone help?
Cheers

**Comment by etienneunai** [comment]
I had this same issue, fixed it by updating [Microsoft Visual C++ Redistributable](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022)

**Comment by planetchili** [comment]
Yes, this package requires the redistributable as a separate installer. In the future me may consider bundling the runtime with the installer.
